scripts/dropins.js

```diff
@@ -12,15 +12,84 @@ import { initializers, Initializer } from '@dropins/tools/initializer.js';
 // Drop-ins
 import * as authApi from '@dropins/storefront-auth/api.js';
 import * as cartApi from '@dropins/storefront-cart/api.js';
+import * as orderApi from '@dropins/storefront-order/api.js';
 
 // Recaptcha
 import * as recaptcha from '@dropins/tools/recaptcha.js';
 
 // Libs
 import { getConfigValue, getCookie } from './configs.js';
+import { getMetadata } from './aem.js';
 
 export const getUserTokenCookie = () => getCookie('auth_dropin_user_token');
 
+const initializeOrderApi = (orderRef) => {
+  initializers.register(orderApi.initialize, {
+    orderRef,
+  });
+};
+
+const redirectTo = (path) => {
+  window.location.href = path;
```

Comment:
Is this function really necessary? Consider then moving to some utils and using it elsewhere (I saw you also use `href = ` in some other file.